### DISCUSSION TITLE
Problems with  delays and no Intercom function in Homekit

### DISCUSSION THREAD

**Discussion by ZisHic** [dgreif/ring#1355] [Q&A]
Hello guys, thanks for the nice Plugin for the Device: „Ring Intercom“. I have successfully integrated it into my HomeKit. I have sometimes  problems with it and i  missing 1 function: When I want to unlock the door, i need  sometimes  2 attempts to open the door or sometimes I don't get a notification. If it does work with a delay, who else has the problem? , and can not 
![IMG_5848](https://github.com/dgreif/ring/assets/150020292/aea856af-4100-4089-9325-4a780d6a6748)
 find the intercom button in the Homekit to talk with the people. What am I doing wrong? Thank you
![IMG_5849](https://github.com/dgreif/ring/assets/150020292/eddaebb6-7ea8-4341-87b5-3e5af3092c66)

**Comment by dgreif** [comment]
Sorry for the confusion, there is no actual button in HomeKit to allow 2-way communication with the intercom. The button you see in Homebridge is used to provide notifications when the button on the intercom is pressed.

Regarding notification issues, please check out https://github.com/dgreif/ring/wiki/Notification-Troubleshooting
